uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -949,6 +990,23 @@ define([
       }
 
       /**
+       * Loads the managees of currently logged in user
+       *
+       * @return {Promise}
+       */
+      function loadManagees() {
+        var self = this;
```

Comment:
To be honest that is fine with me, because we should follow a single convention all over the code. 
```
function loadManagees() {
 
        return Contact.find(self.directiveOptions.contactId)
          .then(function (contact) {
            contact.leaveManagees()
              .then(function (contacts) {
                self.managedContacts = contacts;
              }.bind(this));
          }.bind(this));
      }